### PR TITLE
Dash styles

### PR DESCRIPTION
**Pull request by BenHSmith13**
# Code-review checklist #

  * Documentation
      * [ ] Interfaces describe how other programmers would use them.
      * [ ] Code that is not obvious has documentation describing why
        that code is the way it is.

### CODE REVIEW

File: client/js/_author/components/bank_navigation/bank_list_item.jsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React                    from 'react';
 import Icon                     from './bank_icon';
-import { colors, buttonStyle }  from '../../defines';
 
 // TODO: think about breaking this into smaller components
 export default class BankListItem extends React.Component {
```
Comment (dspencer001) [inline client/js/_author/components/bank_navigation/bank_list_item.jsx]: Without state, is there any reason for this not to be a functional component?

File: client/js/_author/dashboard2.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```
Comment (dspencer001) [inline client/js/_author/dashboard2.html]: What's this file?

Comment (BenHSmith13) [inline client/js/_author/dashboard2.html]: UUuuu, something I forgot to delete...

File: client/js/_author/components/bank_navigation/bank_list_item.jsx (hunk 1 above)
Comment (BenHSmith13) [inline client/js/_author/components/bank_navigation/bank_list_item.jsx]: nope, none at all

File: client/js/_author/components/bank_navigation/bank_list_item.jsx [hunk 3]
```diff
@@ -1,119 +1,59 @@
...
-  }
+      </td>
+    </tr>
+  );
 }
+
+bankListItem.propTypes = {
+  bank: React.PropTypes.shape({
+    displayName : React.PropTypes.shape({}),
+    type        : React.PropTypes.string,
```
Comment (BenHSmith13) [inline client/js/_author/components/bank_navigation/bank_list_item.jsx]: UUUUhhg

**Pull request merged**